composer.json

```diff
@@ -87,7 +87,25 @@
             "Rector\\": ["packages", "rules"],
             "Rector\\Core\\": "src",
             "Rector\\Compiler\\": "utils/compiler/src"
-        }
+        },
+        "files": [
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/DeprecatedTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/InvalidTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/MethodTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/MethodTagValueParameterNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/GenericTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/ReturnTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/TemplateTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/PhpDocTextNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/ParamTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/PhpDoc/VarTagValueNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayTypeNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/ArrayShapeItemNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/GenericTypeNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/CallableTypeNode.php",
+            "vendor/phpstan/phpdoc-parser/src/Ast/Type/CallableTypeParameterNode.php"
+        ]
```

Comment:
This is require to give the fork vendor/phpdoc-parser priority over internal `phpstan.phar` ones. Resp. to give priority `BaseNode` with attributes over none.